sdk/core/System.Net.ClientModel/api/System.Net.ClientModel.net6.0.cs

```diff
@@ -0,0 +1,407 @@
+        public static T? ToNullable<T>(System.Net.ClientModel.Internal.OptionalProperty<T?> optional) where T : struct { throw null; }
+        public static T? ToNullable<T>(System.Net.ClientModel.Internal.OptionalProperty<T> optional) where T : struct { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct OptionalProperty<T>
+    {
+        private readonly T _Value_k__BackingField;
+        private readonly int _dummyPrimitive;
+        public OptionalProperty(T value) { throw null; }
+        public bool HasValue { get { throw null; } }
+        public T Value { get { throw null; } }
+        public static implicit operator T (System.Net.ClientModel.Internal.OptionalProperty<T> optional) { throw null; }
+        public static implicit operator System.Net.ClientModel.Internal.OptionalProperty<T> (T value) { throw null; }
+    }
+    public static partial class PipelineProtocolExtensions
+    {
+        public static System.Net.ClientModel.NullableResult<bool> ProcessHeadAsBoolMessage(this System.Net.ClientModel.Core.MessagePipeline pipeline, System.Net.ClientModel.Core.ClientMessage message, System.Net.ClientModel.RequestOptions requestContext) { throw null; }
+        public static System.Threading.Tasks.ValueTask<System.Net.ClientModel.NullableResult<bool>> ProcessHeadAsBoolMessageAsync(this System.Net.ClientModel.Core.MessagePipeline pipeline, System.Net.ClientModel.Core.ClientMessage message, System.Net.ClientModel.RequestOptions requestContext) { throw null; }
+        public static System.Net.ClientModel.Core.MessageResponse ProcessMessage(this System.Net.ClientModel.Core.MessagePipeline pipeline, System.Net.ClientModel.Core.ClientMessage message, System.Net.ClientModel.RequestOptions requestContext, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public static System.Threading.Tasks.ValueTask<System.Net.ClientModel.Core.MessageResponse> ProcessMessageAsync(this System.Net.ClientModel.Core.MessagePipeline pipeline, System.Net.ClientModel.Core.ClientMessage message, System.Net.ClientModel.RequestOptions requestContext, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+    }
+    public partial class Utf8JsonContentWriter : System.IDisposable
+    {
+        public Utf8JsonContentWriter() { }
+        public System.BinaryData WrittenContent { get { throw null; } }
+        public void Dispose() { }
+        protected virtual void Dispose(bool disposing) { }
+        public void Write(System.Net.ClientModel.Internal.IUtf8JsonContentWriteable content) { }
+    }
+}
+namespace System.Net.ClientModel.Internal.Core
+{
+    public partial class HttpClientPipelineTransport : System.Net.ClientModel.Core.PipelineTransport, System.IDisposable
```

Comment:
In the default scenario where no custom transport is provided, is there a reason this needs to be disposable?